src/modules/launcher/Wox.Infrastructure/Storage/BinaryStorage.cs

```diff
@@ -28,6 +32,15 @@ public BinaryStorage(string filename)
 
         public T TryLoad(T defaultData)
         {
+            // Depending on the version number of the previously installed PT Run, delete the cache if it is found to be incompatible
+            if (_storageHelper.clearCache)
+            {
+                if(File.Exists(FilePath))
+                {
+                    File.Delete(FilePath);
```

Comment:
nit:  maybe we want to log this.   It might be helpful in logs to see a `Log.Info("|BinaryStorage.TryLoad|Cache deleting cached data");`

before 
`Log.Info("|BinaryStorage.TryLoad|Cache file not exist, load default data");`